.env.example

```
@@ -11,11 +11,35 @@ VITE_APP_NAME=HalluciFix
+
+# AWS Cognito Authentication (REQUIRED)
+VITE_COGNITO_USER_POOL_ID=us-east-1_XXXXXXXXX
+VITE_COGNITO_USER_POOL_CLIENT_ID=your_cognito_client_id
+VITE_COGNITO_IDENTITY_POOL_ID=us-east-1:xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+VITE_COGNITO_DOMAIN=your-cognito-domain.auth.us-east-1.amazoncognito.com
+
+# AWS RDS Database (REQUIRED)
+DATABASE_URL=postgresql://username:password@your-rds-endpoint:5432/hallucifix
+DB_HOST=your-rds-endpoint.region.rds.amazonaws.com
+DB_PORT=5432
+DB_NAME=hallucifix
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_NAME key should go before the DB_PORT key

(UnorderedKey)